rasa_sdk/tracing/instrumentation/attribute_extractors.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Dict, Text
+from rasa_sdk.executor import ActionExecutor
+from rasa_sdk.types import ActionCall
+
+
+# This file contains all attribute extractors for tracing instrumentation.
+# These are functions that are applied to the arguments of the wrapped function to be
+# traced to extract the attributes that we want to forward to our tracing backend.
+# Note that we always mirror the argument lists of the wrapped functions, as our
+# wrapping mechanism always passes in the original arguments unchanged for further
+# processing.
+
+
+def extract_attrs_for_action_executor(
+    self: ActionExecutor,
+    action_call: ActionCall,
+) -> Dict[Text, Any]:
+    """Extract the attributes for `ActionExecutor.run`.
+
+    :param self: The `ActionExecutor` on which `run` is called.
+    :param action_call: The `ActionCall` argument.
+    :return: A dictionary containing the attributes.
+    """
+    return {
+        "next_action": action_call["next_action"],
```

Comment:
this could take `None` value according to this type annotation: https://github.com/RasaHQ/rasa-sdk/blob/33b8fa4fd2de67de402c2eaedd1c28c212655f3c/rasa_sdk/types.py#L50